RxSwiftWorkshop/Flights/ViewModel/FlightsViewModelSpec.swift

```diff
@@ -0,0 +1,91 @@
+//
+// Created by Maciej Oczko on 02.05.2017.
+// Copyright (c) 2017 Mobile Academy. All rights reserved.
+//
+
+import Foundation
+import Quick
+import Nimble
+import RxSwift
+import RxTest
+
+@testable import RxSwiftWorkshop
+
+final class FlightsViewModelSpec: QuickSpec {
+    override func spec() {
+        describe("FlightsService") {
+
+            var sut: FlightsViewModel!
+            var schipolService: FakeSchipolService!
+            var iataService: FakeIATAService!
+
+            var scheduler: TestScheduler!
```

Comment:
Not sure whether there's a reason for having IUOs as properties and not just providing the default value. I mean, it's one more allocation, but I think it could be avoided with lazy properties (will check is as soon as I finish pods update)